### PR TITLE
Avoid interferring with installations of upstream software.

### PR DESCRIPTION
**Pull request by Member1221**
This PR updates the dbus endpoint name to match our new domain. There's still more to do to get it working fully independant of upstream.

### CODE REVIEW

**Comment by Member1221** [comment]
btw please squash before merging

**Comment by Member1221** [comment]
This PR is flawed, so i'll redo it when I have time and do stuff properly

File: app/gui/Makefile.am [hunk 1]
```diff
@@ -75,7 +75,7 @@ CLEANFILES = $(gen_sources)
 $(srcdir)/gimpdbusservice.c: $(libappgui_a_built_sources)
 
 $(libappgui_a_built_sources): $(srcdir)/dbus-service.xml
-	$(GDBUS_CODEGEN) --interface-prefix org.glimpse_editor.Glimpse. \
+	$(GDBUS_CODEGEN) --interface-prefix org.glimpse-editor.Glimpse. \
```
Comment (bochecha) [inline app/gui/Makefile.am]: This is wrong, hyphens are an illegal character in the DBus spec.

The recommendation is to replace them by underscores, which is why I did it this way in the first place.

Please don't change those, they are correct as they are.

Comment (Member1221) [reply app/gui/Makefile.am]: I know I messed up which is why I closed this pr, my fork was in a weird limbo state where it wouldn’t sync up properly

**Pull request closed without merging**